config/ci/manager_image_patch.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: apps/v1
```

Comment:
I'm sure these are still needed for testing, but we should really get rid of these duplicate files